brands/shop/newsagent.json

```diff
@@ -11,6 +11,8 @@
   "shop/newsagent|Hudson News": {
     "tags": {
       "brand": "Hudson News",
+      "brand:wikidata": "Q5928787",
+      "brand:wikipedia": "en:Hudson News",
```

Comment:
I’m not sure if these are valid anymore. Hudson News has rebranded every location I’ve been to across the US lately to just Hudson.

Reply:
I don't know that brand by experience, I just found it on Wikidata.

Would this be okay, instead?

```
  "shop/newsagent|Hudson": {
    "matchNames": ["Hudson News"],
    "tags": {
      "brand": "Hudson",
      "brand:wikidata": "Q5928787",
      "brand:wikipedia": "en:Hudson News",
      "name": "Hudson",
      "shop": "newsagent"
    }
  },
```

(Maybe we should change the WIkipedia article too, but that's the less important data. It's currently a redirection anyways.)

Reply:
in 1b5ea1777 I changed it to this:
```js
  "shop/newsagent|Hudson": {
    "matchNames": ["hudson news"],
    "tags": {
      "brand": "Hudson",
      "brand:wikidata": "Q5928682",
      "brand:wikipedia": "en:Hudson Group",
      "name": "Hudson",
      "shop": "newsagent"
    }
  },
```